misk-hibernate/src/main/kotlin/misk/vitess/DockerVitessCluster.kt

```diff
@@ -295,49 +291,34 @@ class DockerVitessCluster(
         "-num_shards=$shardCounts"
     )
 
-    val containerName = "$CONTAINER_NAME_PREFIX-${cluster.name}"
-
     val prefixes = listOf(
         "docker-vitess-testing", // These are the prefixes of banklin/franklin containers
         CONTAINER_NAME_PREFIX // These are misk containers
     )
-    val allContainers = docker.listContainersCmd().exec()
+    val allContainers = docker.listContainersCmd().withShowAll(true).exec()
     val vitessContainers = prefixes.flatMap { prefix ->
       allContainers.filter { container ->
         container.name().startsWith(prefix)
       }
     }
-    // Kill and remove Vitess containers for other apps
-    val (myContainers, otherContainers) =
-        vitessContainers.partition { container -> container.name() == containerName }
-    otherContainers.forEach { container ->
-      logger.info {
-        "Existing Vitess container ${container.name()} for another app found, " +
-            "force killing to free up the port"
-      }
-      docker.removeContainerCmd(container.id).withForce(true).exec()
-    }
 
-    // Kill and remove Vitess containers that aren't running
-    val (runningContainers, deadContainers) =
-        myContainers.partition { container -> container.state == "running" }
-    deadContainers.forEach { container ->
-      logger.info {
-        "Existing Vitess container named ${container.name()} found in " +
-            "state ${container.state}, force removing and restarting"
+    // Kill and remove Vitess containers that doesn't match our requirements
```

Comment:
Is this from the "kill container" PR?

Reply:
I did some fixes to this so that it would automatically restart if the image changed (as I had to do in this PR).